Policies/KeyVault/Azure_KeyVault_NetSec_Disable_Public_Network_Access/Security - Key Vault - DisablePublicNetworkAccess.json

```diff
@@ -0,0 +1,53 @@
+{
+    "mode": "All",
+    "parameters": {
+      "effect": {
+        "type": "String",
+        "metadata": {
+          "displayName": "Effect",
+          "description": "The effect determines what happens when the policy rule is evaluated to match"
+        },
+        "allowedValues": [
+          "Audit",
+          "Disabled",
+          "Deny"
+        ],
+        "defaultValue": "Audit"
+      }
+    },
+    "policyRule": {
+      "if": {
+        "allof": [
+          {
```

Comment:
{
            "field": "type",
            "equals": "Microsoft.KeyVault/vaults"
          }
          
          Let's first add which resource type policy is targetting.

Reply:
Added